qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -84,6 +84,8 @@
             input_dev_bus_type_input1 = virtio
             input_dev_type_input1 = mouse
             vmcoreinfo = yes
+            Win10.i386:
+                mem = 4096
```

Comment:
As we discussed, besides the virtio-fs feature, I think we can not set the mem to 4G forcibly.
Pls correct me if I misunderstand.

Reply:
Because it's in - @all_drivers varians, including virtiofs config, so set the limitation here.

Reply:
> Because it's in - @all_drivers varians, including virtiofs config, so set the limitation here.

yeah, but this limitation just works for virtiofs. so we should indicate it explicitly since other drivers do not.
BTW, the bellow line 189~190:
```
                - with_viofs:
                    Win10.i386:
                        mem = 4096
```
 we also limit the VM mem size to 4G,  it looks like a bit redundant.
Pls let me know your opinions. Thanks.

Reply:
Others driver on Win10.i386 also has memory limitation, and I think currently most of host has memory bigger than 4G, so I think it's ok to set Win10.i386 with 4G memory for all drivers.
For the redundant, as the configs belong to two variants, and the single_driver variant has other drivers. Actually I don't know how to make it more efficient, do you have some good suggestion? 
Thanks.

Reply:
> Others driver on Win10.i386 also has memory limitation, and I think currently most of host has memory bigger than 4G, so I think it's ok to set Win10.i386 with 4G memory for all drivers. For the redundant, as the configs belong to two variants, and the single_driver variant has other drivers. Actually I don't know how to make it more efficient, do you have some good suggestion? Thanks.

I think we could keep the previous config, since the https://github.com/avocado-framework/avocado-vt/pull/3828/files has limited the `vm_mem_limit = 4G` for Win10.i386, we just consider the virtio fs testing, like you configured inside the` - with_viofs:` 
or the feature owner who is responsible for [win_virtio_driver_install_by_installer.cfg](https://github.com/autotest/tp-qemu/pull/3994/files#diff-f17505e969c5502f351d5ca12529922ff28baad6602a08e2ef1ced0ee815151c) , we can listen to her/his opinoin.

Hi @leidwang 
What's your comment about this change?
Thanks.